src/Bookings.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import Search from "./Search.js";
-// import SearchResults from "./SearchResults.js";
-// import FakeBookings from "./data/fakeBookings.json";
+import SearchResults from "./SearchResults.js";
+import FakeBookings from "./data/fakeBookings.json";
```

Comment:
it is convention to use Pascale case (where the first letter is also capitalised) only for JSX components, so this should be `import fakeBookings from "./data/fakeBookings.json";`

Reply:
Thanks Ellie I changed it.